server/handlers.go

```diff
@@ -870,6 +899,8 @@ func (s *Server) headHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	metadata, err := s.GetMetadata(token, filename)
```

Comment:
see first comment: we are doing basically two times the same thing in https://github.com/dutchcoders/transfer.sh/pull/234/files#diff-105fedf355a9fcad82982745cef24f7aR896 and here